core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy

```diff
@@ -21,7 +21,7 @@ l.layout(norefresh:true, permission:app.SYSTEM_READ, title:my.displayName) {
         if (!clouds.isEmpty()) {
             p()
             div(class:"behavior-loading") {
-                l.spinner(text: _("LOADING"))
+                l.spinner(text: _("Loading"))
```

Comment:
why update this place to `Loading` whereas in the other places add translations for `LOADING`?

Reply:
There was only the one instance of this (`core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index_it.properties`) so I changed the variable name too - happy to undo it though and add a separate default translation file.

Reply:
possibly something to follow up with and make them all consistent? Ideally `Loading`

Reply:
Could easily be a reusable control.